overlord/snapstate/snapstate_test.go

```diff
@@ -7314,8 +7336,10 @@ func (s *snapmgrTestSuite) TestStartSnapServicesWithDisabledServicesNowApp(c *C)
 
 	s.runStartSnapServicesWithDisabledServices(c, "hello")
 
+	re := regexp.MustCompile(`.*previously disabled service hello is now an app and not a service\n.*`)
+
 	// check the log for the notice
-	c.Assert(buf.String(), Matches, `.*previously disabled service hello is now an app and not a service\n.*`)
```

Comment:
Out of curiosity, why this change ? It seems the original was just fine? It probably need a regexp flag though, this:
```
c.Assert(buf.String(), Matches, `(?s).*previously disabled service hello is now an app and not a service\n.*`)
```

should work (here and below).

Reply:
Must be merge conflict wrongly solved.

Reply:
Ah, right, now I remember. That works.